R/A_nhdplusTools.R

```diff
@@ -307,10 +307,6 @@ nhdplusTools_data_dir <- function(dir = NULL) {
 }
 
 .onAttach <- function(libname, pkgname) {
-  packageStartupMessage(paste(strwrap(
-    "USGS Support Package:
-    https://owi.usgs.gov/R/packages.html#support"),
-    collapse = "\n"))
   nhdplus_path(default_nhdplus_path, warn = FALSE)
```

Comment:
Oh! I like this a lot... might be worth wrapping in some text: `paste("Default nhdpath is: ", ... )` ?

Reply:
I actually want to deprecate the nhdplus_path thing. It was a nice idea but I don't use it anymore -- do you think it's worth keeping?